src/exp_and_gram.jl

```diff
@@ -95,20 +154,39 @@ function _exp_and_gram_double(Φ0, U0, s)
 end
 
 
-
 """
-    _exp_and_gram_init(A::AbstractMatrix{T}, B::AbstractMatrix{T}, L::LegendreExp{T})
+    _exp_and_gram_chol_init(A::AbstractMatrix{T}, B::AbstractMatrix{T}, L::LegendreExp{T})
```

Comment:
```suggestion
    _exp_and_gram_chol_init!(A::AbstractMatrix{T}, B::AbstractMatrix{T}, L::LegendreExp{T})
```

Reply:
Actually no, the docstring needs to be changed more as the function signature changed

Reply:
Yes, also ```L::LegendreExp{T}``` is from before the code was refactored a couple of months ago, so it needs to go.